### PR TITLE
Update install notes

### PR DESCRIPTION
**Pull request by tomasMizera**
Updated Android and MacOS install steps.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -211,43 +212,66 @@ If you have "error: undefined reference to 'stdout'" or so, make sure that in BU
...
+
+  # (optional, not needed often) add Java to PATH if you need to use other Java version than your default one
+  export PATH=/opt/homebrew/Cellar/openjdk@17/17.0.11/libexec/openjdk.jdk/Contents/Home/bin:$PATH;
+
   cmake \
-    -DCMAKE_BUILD_TYPE=Release \
-    -DQT_ANDROID_ABIS="arm64-v8a;armeabi-v7a" \
-    -DQT_HOST_PATH=$QT_BASE/macos \
-    -DCMAKE_TOOLCHAIN_FILE=$QT_BASE/android_arm64_v8a/lib/cmake/Qt6/qt.toolchain.cmake \
+    -DCMAKE_BUILD_TYPE=Debug \
```
Comment (PeterPetrik) [inline INSTALL.md:266]: Does Debug work ? I always use RelWithDbInfo for mobile apps,  i think on some devices the debug build is not starting.

Comment (tomasMizera) [reply INSTALL.md:266]: I use debug all the time 🤔, didn't know it causes troubles on some machines

Comment (tomasMizera) [reply INSTALL.md:266]: I use release for iOS

Comment (PeterPetrik) [reply INSTALL.md:266]: maybe it is for iOS and I am confused. If you use it is it fine.

Comment (tomasMizera) [reply INSTALL.md:266]: ... to do release build on Android you need all these signing certificates, don't you?

**Pull request merged**